elasticdl/pkg/server/psserver.go

```diff
@@ -0,0 +1,86 @@
+package main
```

Comment:
package main --> package server?

And what about 
elasticdl/pkg/server/psserver.go -->  elasticdl/pkg/ps/server.go